lib/backend/etcdv3/conversion.go

```diff
@@ -91,9 +94,23 @@ func etcdToKVPair(key model.Key, ekv *mvccpb.KeyValue) (*model.KVPair, error) {
 		}
 	}
 
+	if _, ok := v.(*apiv3.GlobalNetworkPolicy); ok {
```

Comment:
don't forget NetworkPolicy

Reply:
Since this is a WIP, I wanted to check one first before I do the other. I'll definitely remember to add it in if this approach looks good, but it seems a little hackey to me and I'm looking for a better way.